test/commands/apex/list/log.test.ts

```diff
@@ -83,66 +83,47 @@ describe('apex:log:list', () => {
   });
 
   it('will list multiple logs', async () => {
-    sandbox.stub(LogService.prototype, 'getLogRecords').resolves(logRecords);
-    const result = await new Log([], config).run();
-    expect(result).to.deep.equal(logRecords);
+    sandbox.stub(LogService.prototype, 'getLogRecords').resolves(structuredClone(logRecords));
+    await new Log([], config).run();
     expect(tableStub.firstCall.args[0]).to.deep.equal([
```

Comment:
had to change the test because the implementation doesn't make all the new columns anymore.

doesn't assert the output structure since the next test does that